src/docs/product/sentry-basics/search/searchable-properties.mdx

```diff
@@ -28,6 +28,12 @@ You can search by event properties in the following [sentry.io](https://sentry.i
 
 When searching on event properties within the **Issues** page, the search will return any issue that has _one or more events_ matching the supplied event properties filter.
 
+### Replay Properties
+
+[Session Replay](/product/session-replay/) objects are not represented within the existing event data tables inside Sentry. As such, they share some common field names, but also have unique fields as well.
+
+You can search by session replay properties on the **Replays** page.
+
```

Comment:
```suggestion

[Session Replay](/product/session-replay/) provides a video-like reproduction of user interactions on a site or web app. All user interactions, including page visits, mouse movements, clicks, and scrolls, are captured, helping developers connect the dots between a known issue and how a user experienced it in the UI.

You can search by session replay properties (indicated with an R in the properties table below) on the **Replays** page.

```
Can we link them to the Replays page in the product?

Reply:
I don't think we can link them into the product because we don't have their org-slug.

Reply:
@ryan953 You can do this: https://sentry.io/orgredirect/organizations/:orgslug/replays/